From 97a3e4ec7c89e0db167093810a3925e3f1e6bdbc Mon Sep 17 00:00:00 2001
From: Lucas Stach <l.stach@pengutronix.de>
Date: Thu, 2 Apr 2015 17:30:52 +0200
Subject: [PATCH 096/136] staging: etnaviv: some final trivial changes to the
 module

- correct license to the proper GPLv2
- add correct author names
- remove double MODULE_DEVICE_TABLE
- update driver date

Signed-off-by: Lucas Stach <l.stach@pengutronix.de>
Signed-off-by: Russell King <rmk+kernel@arm.linux.org.uk>
---
 drivers/staging/etnaviv/etnaviv_drv.c | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/drivers/staging/etnaviv/etnaviv_drv.c b/drivers/staging/etnaviv/etnaviv_drv.c
index 6e5117a..4beca62 100644
--- a/drivers/staging/etnaviv/etnaviv_drv.c
+++ b/drivers/staging/etnaviv/etnaviv_drv.c
@@ -553,7 +553,7 @@ static struct drm_driver etnaviv_drm_driver = {
 	.fops               = &fops,
 	.name               = "etnaviv",
 	.desc               = "etnaviv DRM",
-	.date               = "20130625",
+	.date               = "20150302",
 	.major              = 1,
 	.minor              = 0,
 };
@@ -638,7 +638,7 @@ static struct platform_driver etnaviv_platform_driver = {
 	.remove     = etnaviv_pdev_remove,
 	.driver     = {
 		.owner  = THIS_MODULE,
-		.name   = "vivante",
+		.name   = "etnaviv",
 		.of_match_table = dt_match,
 	},
 };
@@ -666,8 +666,9 @@ static void __exit etnaviv_exit(void)
 }
 module_exit(etnaviv_exit);
 
-MODULE_AUTHOR("Rob Clark <robdclark@gmail.com");
+MODULE_AUTHOR("Christian Gmeiner <christian.gmeiner@gmail.com>");
+MODULE_AUTHOR("Russell King <rmk+kernel@arm.linux.org.uk>");
+MODULE_AUTHOR("Lucas Stach <l.stach@pengutronix.de>");
 MODULE_DESCRIPTION("etnaviv DRM Driver");
-MODULE_LICENSE("GPL");
-MODULE_ALIAS("platform:vivante");
-MODULE_DEVICE_TABLE(of, dt_match);
+MODULE_LICENSE("GPL v2");
+MODULE_ALIAS("platform:etnaviv");
-- 
2.6.1

